nextflow.config

```diff
@@ -39,6 +41,95 @@ params {
   mz_extraction_window = 30
   mz_extraction_window_unit = 'ppm'
   rt_extraction_window = 600
+  extra_rt_extraction_window = 0
+  ion_mobility_window = -1
+  im_extraction_window_ms1 = -1
+  use_ms1_ion_mobility = true
+  matching_windows_only = false
+  irt_mz_extratcion_window = 50
+  irt_mz_extraction_window_unit = 'ppm'
+  irt_im_extraction_window = -1
+  min_coverage = 0.6
+  split_file_input = false
+  use_elution_model_score = false
+  mz_correction_function = 'none'
+  extraction_function = 'tophat'
+  outer_loop_threads = -1
+  ms1_isotopes = 3
+  log = ''
+  debug = 0
+  no_progress = false
+  test = false
+  ms1_im_calibration = false
+  debug_im_file = ''
+  debug_mz_file = ''
+  retentionTimeInterpretation = 'iRT'
+  override_group_label_check = 'linear'
+  force_invalid_mods = false
+  sonar = false
+  useIterativeChauvenet = false
+  RANSACMaxIterations = 1000
+  RANSACMaxPercentRTThreshold = 3
+  RANSACSamplingSize = 10
+  InitialQualityCutoff = 0.5
+  OverallQualityCutoff = 5.5
+  MinPeptidesPerBin = 1
```

Comment:
Please do not make parameters that start with a capital letter. You can see that GitHub syntax highlighting colours these differently and that's because in groovy something that starts with a capital should be a Class and not a variable.